schemas/EiffelActivityCanceledEvent/1.0.0.json

```diff
@@ -1,24 +1,25 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
+  "javaType": "com.ericsson.eiffel.remrem.semantics.events.EiffelActivityCanceledEvent",
+  "extendsJavaClass": "com.ericsson.eiffel.remrem.semantics.events.Event",
```

Comment:
This is what you might call a "soft dependency" from the protocol to a specific implementation. Which is really a bad thing. On the other hand, the pragmatic reason is obvious.

When I look at this now (sorry for being a bit back and forward on this), the way to do this would be:
1) Keep javaType and extendsJavaType out of the schema definitions.
2) As a build step in Remrem, pull in schemas from Eiffel and inject necessary properties.
3) Use those ad-hoc modified schemas in the Remrem service.

That way you don't end up duplicating the schemas, and Eiffel doesn't have to know about the name of a class in some particular serialization implementation.